core/src/magic_string.rs

```diff
@@ -631,6 +631,88 @@ impl MagicString {
     SourceMap::new_from_decoded(decoded_map)
   }
 
+  pub fn _move(&mut self, start: i64, end: i64, index: i64) -> Result<&mut Self> {
```

Comment:
Would you add some doc to this method and change it to `move`?

Reply:
Yes, I will add it.